doc/api/cli.md

```diff
@@ -2937,6 +2937,10 @@ The following values are valid for `mode`:
 
 ### `--use-system-ca`
 
+<!-- YAML
+added: v23.8.0
```

Comment:
Did you mean to include an entry for v23.9.0?

Reply:
@joyeecheung done, could you PTAL?